app/Models/Sprint.php

```diff
@@ -81,4 +82,16 @@ public function getLastUpdatedDate(): Carbon
     {
         return $this->lastUpdated;
     }
+
+    public function containsToday(): Bool
+    {
+        if (empty($this->startDate)  or empty($this->endDate)) {
```

Comment:
I'd use the getters here getStartDate() and getEndDate().  Also, does `or` actually work?  I'd replace it with `||` even if it does.

Reply:
Good catch! I'll update those to the getters...

For the or logic, yes it does seem to work. I wasn't sure at the time which was valid php at the time, and a quick google search I found told me this:
> The || and or operators return the same result. The only difference between the || and or operators are their precedences. The or operator has higher precedence than the || operator.

Considering that I just picked one semi-randomly! If you prefer `||` I don't have a problem switching to that!